base/testfiles-TU/github-1380.luatex.tlg

```diff
@@ -0,0 +1,6 @@
+This is a generated file for the LaTeX2e validation system.
+Don't change this file in any respect.
+luaotfload | db : Font names database loaded from ../luaotfload-names.luc.gz
```

Comment:
should we try to supress line 3 or would it always come out like this? E.g., move the test after \begin{document}?

Reply:
I think it's just luaotfload being chatty, every luatlatex has that line somewhere in the log, it just got picked up here as it's the first typesettung done (and inconvenient to pre-use the font before `\START` as this was in the preamble, and the format wasn't quite the same as xetex anyway so it would still have needed a `.luatex.tlg` even without this line

Reply:
if it is always the same line then it doesn't make a difference I agree, I was just wondering if it might change, in which case one could have moved the hyphenation test after begin document (or so I thought) and use \START then

Reply:
i adjusted to avoid the luaotfload message